.buildkite/pipeline.yml

```diff
@@ -58,3 +57,6 @@ steps:
       - label: "Water conservation"
         command: "julia --color=yes --project=experiments experiments/standalone/Soil/water_conservation.jl"
         artifact_paths: "experiments/standalone/Soil/water_conservation*png"
+  - wait
+
+  - command: "compact_depot"
```

Comment:
does this cache the artifacts from this job on the cluster? do we need a DepotClear for ClimaLSM in buildkite now?

Reply:
it's different than the shared depot we use in ClimaAtmos: basically in this job we have 2 depots: the first 1 is "clean", and the 2nd one just contains artifacts. We install everything into the 1st one, but at the end of each build, we move all the artifacts to the 2nd one

Reply:
so then do these 2 depots persist across jobs so they can be reused?

Reply:
also, will we ever need to clear these depots or do any other manual management of them?

Reply:
the 2nd one persists, the 1st one does not

Reply:
> also, will we ever need to clear these depots or do any other manual management of them?

I hope not, but I can't say for sure.